Tasks/AzureContainerAppsV0/src/ContainerAppHelper.ts

```diff
@@ -1,10 +1,22 @@
 import * as tl from 'azure-pipelines-task-lib/task';
+import * as path from 'path';
+import * as os from 'os';
 import { CommandHelper } from './CommandHelper';
+import { Utility } from './Utility';
 
-const ORYX_CLI_IMAGE: string = 'cormtestacr.azurecr.io/oryx/cli:latest';
-const ORYX_BUILDER_IMAGE: string = 'cormtestacr.azurecr.io/builder:latest';
+const ORYX_CLI_IMAGE: string = 'mcr.microsoft.com/oryx/cli:builder-debian-buster-20230208.1';
+const ORYX_BUILDER_IMAGE: string = 'mcr.microsoft.com/oryx/builder:20230208.1';
+const AGENT_OS: string = tl.getVariable('AGENT.OS');
```

Comment:
isn't better to use [os.platform()](https://nodejs.org/docs/latest-v10.x/api/os.html#os_os_platform) from nodejs?

Reply:
I'm personally not familiar with the major differences (if any) between using `os.platform()` and the [`AGENT.OS` environment variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#agent-variables-devops-services) set by Azure DevOps -- if there is any guidance around which to use, please feel free to let me know.

I checked the repository and saw that only a couple of tasks are using `AGENT.OS` only for telemetry purposes, and a fair number of tasks are using `os.platform()` for logic purposes, so I'll go ahead and update our task to use `os.platform()` to conform with patterns established by others in this repository.